pkg/tcpip/stack/nic.go

```diff
@@ -482,6 +482,16 @@ func (n *NIC) DeliverNetworkPacket(linkEP LinkEndpoint, remote, _ tcpip.LinkAddr
 
 	src, dst := netProto.ParseAddresses(vv.First())
 
+	// TODO: Handle the packet according to RFC.
```

Comment:
Can you include a link to an issue? If there isn't an appropriate existing issue, feel free to file a new one.

Reply:
added https://github.com/google/gvisor/issues/1085